static/src/javascripts/test/spec/common/commercial/gustyle/gustyle.spec.js

```diff
@@ -0,0 +1,65 @@
+import Injector from 'helpers/injector';
+import $ from 'common/utils/$';
+import fastdom from 'fastdom';
+import fixtures from 'helpers/fixtures';
+
+var Sut, // System under test
```

Comment:
it is not convention here. I find it easier to understand which module I am actually testing, especially when you are working in multiple tests.